src/ansys/fluent/core/fluent_connection.py

```diff
@@ -214,10 +214,10 @@ def __init__(
         )
 
         self.journal = Journal(self.scheme_eval)
+        self.version = self.scheme_eval.string_eval("(cx-version)")
```

Comment:
Has this been pulled out of the try block deliberately?

Reply:
> @hpohekar I've incorporated the fix in #1182, you may close this PR.

@mkundu1 Yes, sure. Thank you.

Reply:
> Has this been pulled out of the try block deliberately?

@seanpearsonuk  Yes, but later I saw your message in teams that we want to keep current behavior. So this change is not useful now.